### ISSUE TITLE
Chrome 51 weird permission problem

### ISSUE THREAD

**Issue by ppp0** [cargomedia/puppet-packages#1398]
Reported by @vogdb in https://github.com/cargomedia/janus-gateway-js/pull/67#issuecomment-234229698

TL;DR: All works fine when changing the downloaded chrome-linux dirtree owned by vagrant, the user under which test scripts will run - 

Added [this temporary workaround](https://github.com/cargomedia/janus-gateway-js/issues/69) which will be removed asap

Suspected:
* Installation issue, although we recursively open as much as supposedly needed
* tree for developers, so probably not completely bug-free
* some yet unknown chrome dependency, .file? .dir? profiledir?

For ubuntu there ia a [PPA repository](https://en.wikipedia.org/wiki/Personal_Package_Archive): https://launchpad.net/ubuntu/%2Bsource/chromium-browser - v51 is for Xenial only

Adding Xenial support to the Chromium module is a good idea anyway... Should be quick

**Comment by kris-lab** [comment]
Closing in favor https://github.com/cargomedia/puppet-packages/issues/1432

**Comment by njam** [comment]
Closing in favor of https://github.com/cargomedia/janus-gateway-js/issues/98
